LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2021 Matthew Feickert
+Copyright (c) 2021-2022 Matthew Feickert
```

Comment:
Thanks, but I have yet to see any legal argument that bumping the copyright year in LICENSE files does anything and from my experience in ATLAS it just adds pointless noise. Given that Google doesn't update their copyright years (e.g. [JAX](https://github.com/google/jax/blob/fd3a72dd1fc1112b698c9f74ee9720d78280e98d/benchmarks/api_benchmark.py)) I think its safe to say everyone in HEP can stop doing this. :) 

I truly appreciate the sentiment though.

Reply:
Yeah, I have seen both schools. I do update years in my repos, and LHCb does as well for its full stack. I don't really believe there is a legal consequence as long as the copyright exists with name(s) associated.